stable/coredns/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: coredns
-version: 1.5.0
-appVersion: 1.5.0
```

Comment:
you only need to increase appversion if the app itself changes. (I've rebased my changes in https://github.com/helm/charts/pull/13310 and will fix this)